src/tests/ftest/util/server_utils.py

```diff
@@ -467,9 +467,9 @@ def support_collect_log(self, **kwargs):
         cmd = DaosServerCommand(self.manager.job.command_path)
         cmd.run_user = "daos_server"
         cmd.debug.value = False
-        cmd.config.value = get_default_config_file("server")
```

Comment:
I think this should work - we could also do
```
kwargs['config'] = get_default_config_file("server")
And let the `set_command` below continue normally
```

Reply:
Isn't that the same thing? `f(a=foo, **kwargs)` is essentially the same as `kwargs['a']=foo; f(**kwargs)` I would have thought

Reply:
Yes, the same functionally :) No need to change